### DISCUSSION TITLE
Add action button to the Activity View title bar

### DISCUSSION THREAD

**Discussion by pankajparashar** [microsoft/vscode-discussions#2454] [Extension Development QnA] [answered]
Is there a VSCode API to add action button to the View Container title section, similar to how Github Copilot Chat VSCode extension does? For reference, checkout the highlighted red box in the screenshot, which has a New Chat button, Historical chats, and more options in the dropdown menu.

![image](https://github.com/user-attachments/assets/3ec4f1ed-9282-49e3-bbcd-27ebd263c28c)

**Comment by alefragnani** [comment, marked as answer]
Hi @pankajparashar ,

If your `viewContainer` has only one _visible_ `view`, the commands you added to that `view` (using the `view/title` contribution point) will be moved to that location. 

The `viewContainer` itself, on the other hand, does not have an API/contribution point to add commands to it.

Hope this helps
